plugins/modules/facts.py

```diff
@@ -561,7 +583,8 @@ def main():
     """main entry point for module execution
     """
     argument_spec = dict(
-        gather_subset=dict(default=['!config'], type='list')
+        gather_subset=dict(default=['!config'], type='list'),
+        verbose_config=dict(default=True, type='bool')
```

Comment:
```suggestion
        verbose_config=dict(default=True, type='bool'),
```
So the next diff will be smaller ;-)